### DISCUSSION TITLE
Only source code listed in assets

### DISCUSSION THREAD

**Discussion by roboticdreaming** [OnionUI/Onion#906] [General]
For Onion V4.2.0.1-beta  (https://github.com/OnionUI/Onion/releases/tag/latest) only the source code is in the assets?  I'm trying to install for the first time on a new device.

**Comment by Aemiii91** [comment]
https://github.com/OnionUI/Onion/issues/896

https://github.com/OnionUI/Onion/discussions/747#discussioncomment-6177794

https://github.com/OnionUI/Onion/discussions/747#discussioncomment-6178760
